care/facility/api/viewsets/asset.py

```diff
@@ -308,3 +310,24 @@ def get_queryset(self):
                 | Q(to_location__facility__id__in=allowed_facilities)
             )
         return queryset
+
+
+class AssetServiceFilter(filters.FilterSet):
+    qr_code_id = filters.CharFilter(field_name="asset__qr_code_id")
+    external_id = filters.CharFilter(field_name="asset__external_id")
+
+
+class AssetServiceViewSet(ListModelMixin, RetrieveModelMixin, GenericViewSet):
+    queryset = (
```

Comment:
Are we okay showing service history without any authZ

Reply:
@vigneshhari If I remember correctly, the asset details are publicly available and are shown on the state dashboard, in that case it should be fine to make their service history public too, right? 

cc: @gigincg

Reply:
@vigneshhari Aparna confirmed to have it publicly visible.